index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+<body>
+  <section class="section-header flex gen-padding">
+    <a href="#" class="logo-box">
+      <img class="logo" src="img/logo.png" alt="logo">
+    </a>
+    <a href="#" class="menu-box">
+      <img class="menu-icon" src="img/menu.png" alt="menu icon">
+    </a>
+  </section>
+  <section class="section-headline flex gen-padding">
+    <h1 class="headline-primary">
+      Hey There. I’m Reza.
+    </h1>
+    <h2 class="headline-support">
+      I'm a Software Developer
+    </h2>
+    <p class="headline-description">
+      I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hesitate to contact me.
+    </p>
+    <div class="social-box flex">
+      <a href="#">
+        <img src="img/github.svg" alt="github icon">
+      </a>
+      <a href="#">
+        <img src="img/in.svg" alt="in icon">
+      </a>
+      <a href="#">
+        <img src="img/Vector.svg" alt="vector icon">
+      </a>
+      <a href="#">
+        <img src="img/twitter.svg" alt="twitter icon">
+      </a>
+      <a href="#">
+        <img src="img/m.svg" alt="m icon">
+      </a>
+    </div>
```

Comment:
- Hey, great job so far! however, to enforce good use of the html5 semantic tag, the content here is a group of list items, so you should use a `ul` as the container with an `li` to hold each `a` tags that wrap each `img`
```
<ul>
  <li><a href=""><img src="" alt=""></a></li>
</ul>
```